app/assets/stylesheets/typography/_base.scss

```diff
@@ -1,16 +1,91 @@
+$primary-font-stack: 'MuseoSans', 'Lato', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
+
 body {
-  @include sc-responsive-typography;
+  color: var(--color-scale-gray-6);
   font-family: $primary-font-stack;
-  min-width: $app-min-width;
+  font-size: $body-font-size;
+  line-height: $lh-default;
+  min-width: 320px;
+
+  h1 {
+    @include h1-responsive;
+  }
+
+  h2 {
+    @include h2-responsive;
+  }
```

Comment:
This is the closest equivalent of the `sc-responsive-typography` mixin from style_closet using Primer mixins.